### PR TITLE
Remove image-registry doc support in download-requirements (#3260)

### PR DESCRIPTION
**Pull request by sbbroot**
* Remove image-registry doc support in download-requirements

### CODE REVIEW

File: ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py [hunk 1]
```diff
@@ -71,6 +72,9 @@ def __log_info_summary(self):
 
         lines.append('-' * self.__LINE_SIZE)
 
+        if self.full_download:
+            lines.append('--full-download flag used, downloading all available resources...')
```
Comment (tomasz-baran) [inline ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py]:
```suggestion
            lines.append('--full-download flag used, downloading all available requirements...')
```

File: ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py [hunk 2]
```diff
@@ -106,6 +110,8 @@ def __create_parser(self) -> ArgumentParser:
 
         parser.add_argument('--manifest', '-m', metavar='MANIFEST_PATH', type=Path, action='store', dest='manifest',
                             help='manifest file generated by epicli')
+        parser.add_argument('--full-download', action='store_true', dest='full_download',
+                            help='download all available resources')
```
Comment (tomasz-baran) [inline ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py]:
```suggestion
                            help='download all available requirements')
```

**Comment by seriva** [comment]
Besides the suggestion resources > requirements I have no further comments.

**Review by seriva — DISMISSED** [review]

**Review by tomasz-baran — DISMISSED** [review]

File: ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py [hunk 3]
```diff
@@ -207,17 +213,18 @@ def __add_args(self, argv: List[str]):
...
-        self.is_log_file_enabled = False if args['no_logfile'] else True
-        self.dest_manifest = args['manifest'] or None
         self.verbose_mode = True if self.log_level == logging.DEBUG else args['verbose']
 
         # offline mode
         self.rerun = args['rerun']
         self.pyyaml_installed = args['pyyaml_installed']
 
-    def __print_parsed_manifest_data(self, requirements: Dict[str, Any], manifest: Dict[str, Any]):
+    def __get_parsed_manifest_data_output(self, requirements: Dict[str, Any], manifest: Dict[str, Any]) -> str:
```
Comment (tomasz-baran) [inline ansible/playbooks/roles/repository/files/download-requirements/src/config/config.py]: SonarCloud reports that `requirements` parameter is unused, how about removing it?

**Review by seriva — DISMISSED** [review]

**Review by tomasz-baran — DISMISSED** [review]

**Review by seriva — DISMISSED** [review]

**Review by tomasz-baran — DISMISSED** [review]

**Review by seriva — DISMISSED** [review]

**Review by seriva — DISMISSED** [review]

File: docs/changelogs/CHANGELOG-2.0.md [hunk 4]
```diff
@@ -10,6 +10,7 @@
 - [#3232](https://github.com/epiphany-platform/epiphany/issues/3232) - Use tarball format insted of legacy for downloading Docker images
 - [#3238](https://github.com/epiphany-platform/epiphany/issues/3238) - Make OpenSearch Dashboards user configurable
 - [#3241](https://github.com/epiphany-platform/epiphany/issues/3241) - Simplify APT commands flow in download-requirements.py
+- [#3260](https://github.com/epiphany-platform/epiphany/issues/3260) - Add `--full-download` flag to download-requirements
```
Comment (przemyslavic) [inline docs/changelogs/CHANGELOG-2.0.md]: Changelog

**Review by seriva — DISMISSED** [review]

**Review by przemyslavic — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**